package.json

```diff
@@ -45,5 +45,8 @@
     "shiki": "^0.2.7",
     "shiki-twoslash": "^0.8.1",
     "tib": "^0.7.4"
+  },
+  "dependencies": {
+    "remark-gfm": "^1.0.0"
```

Comment:
This dependency should be added in the `package.json` of the content package located here https://github.com/nuxt/content/blob/dev/packages/content/package.json.